awx/main/migrations/0060_v350_inventory_update_org_host_limit_error.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Might be nice to squash these two migration files together since it's one PR.

Reply:
Yeah, I'll do it when that other PR lands.